tests/layer_tests/common/layer_test_class.py

```diff
@@ -60,6 +60,8 @@ def _test(self, framework_model, ref_net, ie_device, precision, ir_version, temp
 
         if use_new_frontend:
             mo_params["use_new_frontend"] = True
+        else:
+            mo_params["use_legacy_frontend"] = True
```

Comment:
I suggested this line to enable triggering tests for legacy frontend, but it also has impact on default frontend for layer tests. I think we should set `use_new_frontend` default `True`, to keep testing new frontend as default. 
https://github.com/openvinotoolkit/openvino/blob/9a2f2e446189d396fd27dbc6b7d67520ae00b47b/tests/layer_tests/common/layer_test_class.py#L33